Directory.Build.targets

```diff
@@ -20,4 +20,15 @@
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+  <!--
+    All source inputs to the compiler should be generated before BeforeCompile target. Sdk is not
```

Comment:
Shouldn't this property always be needed? It seems like the comment needs to be on the ItemGroup that is appending to `EmbeddedFiles`

Reply:
No, we are just computing the path of generated file here and then including it that path in embedded files. Bu ti think the comment i think still should be moved to itemgroup

Reply:
I meant that even when the issue is fixed, we will still need to set `GeneratedAssemblyInfoFile`, correct?

Reply:
@safern why do we need the property?

Reply:
You're right. I got confused because of libraries that we write more stuff to that file and define our own property.